python/.pre-commit-config.yaml

```diff
@@ -95,3 +69,9 @@ repos:
           ]
         additional_dependencies:
           - tomli==2.0.1
+  - repo: https://github.com/astral-sh/ruff-pre-commit
```

Comment:
Should we move ruff more to the top? I like it to run it first since it is so fast.

Reply:
I think we can remove this one now